modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -195,6 +196,9 @@ export default class TileLayer extends CompositeLayer {
   }
 
   renderLayers() {
+    if (!this.props.visible) {
+      return null;
```

Comment:
I don't think this is right. `visible` is recommended as an approach to toggle layers without the perf impact of removing/re-adding them. No other composite layer removes its sub layers when visibility changes.